### DISCUSSION TITLE
Move from SD to SSD z2m starts but no communication to devices Pi4

### DISCUSSION THREAD

**Discussion by joene-nl** [Koenkk/zigbee2mqtt#5254] [General]
Hi,

I moved from an SD card Pi4 to an Pi4 with SSD. Restored my snapshots back to the instance running on the SSD. Everything works again except Zigbee2MQTT. The addon starts but nothing happens when i try turning off or on lights. In the logging i see the message
"Data request failed with error: 'No network route' (205)"

I am running on a zzh! CC2652R stick which is working fine when the PI4 is running on SD. Tried connecting the stick using an extension cable cause of the interference issues with a Pi4, but I can't figure out what is going wrong. Should I maybe create a new pan_id and repair all my devices?

I attached the log.txt 
[log.txt](https://github.com/Koenkk/zigbee2mqtt/files/5675315/log.txt)

Curious to hear any thoughts on this.

Thanks in advance

**Comment by Koenkk** [comment]
Have you tried to power cycle the devices? (e.g. remove bulb from mains for a few seconds).

**Reply by joene-nl** [reply]
I already did that. I managed to fix it with powering off home assistant and waiting for 30 min so the lights go in some sort of panic mode like z-wave does. After that I powered HA back up and everything works now!


[https://community.home-assistant.io/t/healing-zigbee-network/135528]()
